CreeDictionary/DatabaseManager/test_db_builder.py

```diff
@@ -9,7 +9,7 @@
 
 from tqdm import tqdm
 
-from profiling.timing import timed
+from utils.profiling import timed
```

Comment:
old `profiling` directory is deleted except for one utility - the `timed` decorator.